### PR TITLE
Test for the snapshot created after installation (fate#317973)

### PR DESCRIPTION
**Pull request by ancorgs**
The functionality is already in TW. So ready to merge

### CODE REVIEW

File: tests/console/installation_snapshots.pm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    become_root();
+
+    script_run("snapper list | tee /dev/$serialdev");
+    # Check if the snapshot called 'after installation' is there
+    $self->result('fail') unless wait_serial("after installation", 5);
```
Comment (aaannz) [inline tests/console/installation_snapshots.pm]: btw. $self->result setting is unnecessary here since wait_serial automatically updates it.

**Comment by imobach** [comment]
This test is ready to be merged. I've tried manually with Tumbleweed snapshot 20150610.

File: tests/console/installation_snapshots.pm [hunk 2]
```diff
@@ -0,0 +1,16 @@
+use base "consoletest";
+use testapi;
+
+sub run() {
+    become_root();
+    script_run("snapper list | tee /dev/$serialdev");
+    # Check if the snapshot called 'after installation' is there
+    wait_serial("after installation", 5);
```
Comment (nilxam) [inline tests/console/installation_snapshots.pm]: I don't get where is the "after installation" from?

**Comment by nilxam** [comment]
LGTM

**Comment by nilxam** [comment]
just a record, this requires snapper version 0.2.7

**Pull request merged**